modules/nodes-about-bare-metal-hosts-and-nodes.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-about-bare-metal-hosts-and-nodes_{context}"]
+= About bare metal hosts and nodes
+
+In order to provision a bare metal host as a node in your cluster, you must first create a `machineset` CR object that corresponds to the bare metal host hardware. Bare metal host machine sets describe infrastructure components specific to your configuration. You apply specific Kubernetes labels to these machine sets and then update the infrastructure components to run on only those machines.
+
+When a `machineset` CR has been created for your hardware host, then create a `machine` CR object. `machine` CR's are used by {product-title} to create the bare metal node that corresponds to the host as specified in the `machineset` CR. Bare metal nodes are created automatically from the relevant `machine` CR.
```

Comment:
```suggestion
When a `machine` CR has been created for your hardware host, then create a `machine` CR object. `machine` CR's are used by {product-title} to provision the bare metal host that corresponds to the host as specified in the `machineset` CR. Bare metal nodes are created automatically from the relevant `machine` CR.
```

Reply:
What about:

When a `machine` container resource definition (CRD) has been created for your {op-system} hardware host, then create a `machine` CR object. A `machine` CR is used to provision the bare metal host as specified by the `machineset` CR. A bare metal node is created automatically from the relevant `machine` CR. `machine` CR's are created automatically when you scale the number of instances of a `machineset`.

Reply:
I've applied your suggestion @andybraren.